test/views/wallet/ConfigureMobilecoindView/LeaveMobilecoindRunning.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import useMobilecoindConfigs from '../../../../app/hooks/useMobilecoindConfigs';
+import LeaveMobilecoindRunning from '../../../../app/views/wallet/ConfigureMobilecoindView';
+import renderSnapshot from '../../../renderSnapshot';
+
+jest.mock('../../../../app/hooks/useMobilecoindConfigs');
+function setupComponent(isLeaveMobilecoindRunning: boolean) {
```

Comment:
Nit, I prefer a space after line 10.

Reply:
-- The rest of the spacing in this file is, more or less, preference. I won't worry too much about it. But feel free to copy my style if you want to keep them uniform. Maybe later I'll add some more complex rules to the linter.